src/MSBuildLocator/MSBuildLocator.cs

```diff
@@ -69,6 +69,16 @@ public static void RegisterInstance(VisualStudioInstance instance)
             if (instance == null)
                 throw new ArgumentNullException(nameof(instance));
 
+            RegisterMSbuildPath(instance.MSBuildPath);
+        }
+
+        /// <summary>
+        ///     Add assembly resolution for Microsoft.Build core dlls in the current AppDomain from the specified
+        ///     path.
+        /// </summary>
+        /// <param name="msbuildPath"></param>
+        public static void RegisterMSbuildPath(string msbuildPath)
```

Comment:
Should the casing be `RegisterMSBuildPath` with a capital **B**? That matches other APIs here.